### PR TITLE
M test

### PR DESCRIPTION
**Pull request by mgmilton**
Add's test's for categories and ensures that categories are being created.

### CODE REVIEW

File: app/views/items/new.erb [hunk 1]
```diff
@@ -1,9 +1,7 @@
 <form action="/items" method="post">
   <p>Enter a new item:</p>
-  <input type='text' name='item[name]'/><br/>
-  <input type='integer' name='item[id]'/><br/>
-  <input type='description' name='item[description]'/><br/>
-  <input type='decimal' name='item[price]'/><br/>
-  <input type='text' name='item[image]'/><br/>
+  <input type='text' name='item[name]' value= "Name"/><br/>
+  <input type='description' name='item[description]' value= "Description"/><br/>
+  <input type='decimal' name='item[price]' value= "Price"/><br/>
```
Comment (zpalmquist) [inline app/views/items/new.erb:5]: So for this, I found a really nice way to make those boxes explicit. 

by doing 
`<input type='text' name='item[name]' value= "Name" placeholder="Enter new name here"/><br/>`

we get that text with the placeholder element inside the box, when the user clicks on the box, that goes away. its like the google search bar or something.

File: app/views/merchants/index.erb [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 <a href="/merchants/new">Create a merchant</a>
 <% @merchants.each do |merchant| %>
-<div class="merchant_<%= merchant.id %>">
```
Comment (zpalmquist) [inline app/views/merchants/index.erb:9]: I kind of prefer this. or "merchant-<%etc....>

File: app/views/items/new.erb (hunk 1 above)
Comment (mgmilton) [reply app/views/items/new.erb:5]: dig it. let's implement it in the next pull request.

File: app/views/merchants/index.erb (hunk 2 above)
Comment (mgmilton) [reply app/views/merchants/index.erb:9]: Works for me.

**Review by zpalmquist — APPROVED** [review]
I'm still approving but just peep my comment and lmk what you think.

File: app/views/merchants/index.erb (hunk 2 above)
Comment (zpalmquist) [reply app/views/merchants/index.erb:9]: so this is now in master but I'll just keep it the way it is on my PR once i have merge conflicts. If I realized we could chat about it before I approved or anything I would have. Next time.

**Pull request merged**